ParabolInc/parabol
# chore: Replace TeamHealth answer emojis with their text representation

# Description

Fixes #8402 
To avoid conversion complications between databases.

## Demo

no demo, looks same as before

## Testing scenarios

- on **master** create a retro with team health
- checkout **PR** and create a new retro
- verify voting, revealing, summary and email summary look fine for old and new retro

## Final checklist
